### DISCUSSION TITLE
Additional clinical annotator for NEW variants in positions with known pathogenic variants

### DISCUSSION THREAD

**Discussion by clinicalngs** [KarchinLab/open-cravat#292] [Usability & Feature Requests]
Hi,
I work with NGS data in a clinical research setting, and we have some unresolved samples. In such cases, I use OC and appreciate its flexibility, operation simplicity and exploratory value. 

I wonder if it is possible to introduce an additional clinical annotator to "catch" positions where the variant detected in a sample is different from the clinically annotated variant in the Clinvar database (e.g. has the same rs#), flagging its "elevated attention" status. It helps to expose new or unevaluated variants, potentially pathogenic, and it can save time with assessments. This is similar to something already available in OC - Clinvar ACMG, which may contain a different variant than the one present in the sample at that position. I am aware that standard in silico predictors of potential impact are useful, but sometimes they come short (e.g. for missense variants, intronic). Other annotators have such ability (e.g. Varsome, Golden Helix, CLC, Franklin, etc.) - some may actually ask which is the found variant, if looking at them individually. 

Recently I got this variant NM_014244.5(ADAMTS2):c.688+1G>**T** (splice), chr5:179272910, HG38. It is not in the official Clinvar database.  However, NM_014244.5(ADAMTS2):c.688+1G>**A** (splice), chr5:179272910, HG38 is evaluated in Clinvar, and it is likely pathogenic. Both have the same rs, and are listed in gnomad  and NCBI (https://www.ncbi.nlm.nih.gov/snp/rs1182409555.)
ACMG algorithm for c.688+1G>T suggests it is also likely pathogenic. 

It would be nice to select such positions with a dedicated filter. I looked for a workaround solution but could not find one. Perhaps you may have an alternative. Any suggestion is kindly appreciated. 

Thank you for such a wonderful software,
Consuel

**Comment by jasminebro** [comment]
Hi @clinicalngs! Thank you so much for your feature request. I will take this back to our IT team and get their feedback and discuss. On another note, your use case is very exciting to hear about and we are happy that OpenCRAVAT has been useful to you in a clinical setting!

If you don't mind, please take 2 minutes to fill out our user feedback survey. We love this type of feedback and would love to keep the discussion going.

[OpenCRAVAT User Feedback](https://docs.google.com/forms/d/e/1FAIpQLSeD00dJl-79E_J1X0n3XhQHWEbQB-cjd-osLr0JXRizbbeeHA/viewform)
